plugins/flashrom/fu-reset-cmos.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Sean Rhodes <sean@starlabs.systems>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+#include "config.h"
+
+#include <libflashrom.h>
+#include <sys/io.h>
+#include <unistd.h>
+
+#include "fu-reset-cmos.h"
+#include "glib.h"
+
+static bool
```

Comment:
as this is a glib project, this should be `static gboolean` not `static bool`

Reply:
Done